motr/obj.c

```diff
@@ -261,8 +261,15 @@ static void obj_namei_cb_launch(struct m0_op_common *oc)
 	m0_sm_group_unlock(&op->op_entity->en_sm_group);
 
 	rc = m0__obj_namei_send(oo);
-	if (rc == 0)
+	if (rc == 0) {
 		m0_sm_move(&op->op_sm, 0, M0_OS_LAUNCHED);
+	} else if (rc == M0TR_COB_LOOKUP_SKIPPED) {
+		/* This is M0_EO_GETATTR and op state is already
```

Comment:
We can end up here in case of `(cr->cr_opcode == M0_EO_CREATE && obj->ob_entity.en_app_type != M0_OOF_MOTR_APP)` as well, not just `M0_EO_GETATTR`

Reply:
Agreed. Will change that comment.